src/tests/notes.rs

```diff
@@ -655,5 +656,18 @@ fn commitments() {
             &mut result
         ));
         assert_eq!(&result, &tv.note_cm);
+
+        // Compute nullifierand compare with test vector
```

Comment:
s/nullifierand/nullifier and/

Reply:
Fixed typo and pushed (and this test file notes.rs is already testing against vectors from Python)